pysite/views/api/bot/tag.py

```diff
@@ -0,0 +1,71 @@
+# coding=utf-8
+__author__ = 'Ferret Moles'
+
+from flask import g, jsonify, session
+
+import rethinkdb
+
+from pysite.base_route import APIView
+
+
+class TagView(APIView):
+    path = '/tag'
+    name = 'tag'
+    table = 'tag'
+
+    def __init__(self):
+
+        # make sure the table exists
+        conn = g.db.get_connection()
+        try:
+            rethinkdb.db(g.db.database).table_create(self.table, {'primary_key': 'tag_name'}).run(conn)
+        except rethinkdb.RqlRuntimeError:
+            print(f'Table {self.table} exists')
+        conn.close()
+
+    def get(self):
+
+        rdb = rethinkdb.table(self.table)
+        tag_name = session.get('tag_name')
```

Comment:
This uses server-side cookie-associated values, I don't think `session` is what you wanted at all.